src/preamble.js

```diff
@@ -2032,4 +2032,343 @@ Module['FS_createPreloadedFile'] = FS.createPreloadedFile;
 var cyberDWARFFile = '{{{ BUNDLED_CD_DEBUG_FILE }}}';
 #endif
 
+#if BINARYEN
+function integrateWasmJS(Module) {
```

Comment:
Also the previous location was nonstandard in that it was before even pre-js files, which confused people. So good to fix for that reason too. The previous location was there just because it was an easy hack in the very early days.